### ISSUE TITLE
Alluxio initialization action fails with alluxio_root_ufs_uri pointed to GCS

### ISSUE THREAD

**Issue by aniket486** [GoogleCloudDataproc/initialization-actions#667]
@madanadit 
We get errors as follows:
```
2019-11-07 21:43:45,131 WARN  UnderFileSystem$Factory - Failed to create UnderFileSystem by factory alluxio.underfs.gcs.GCSUnderFileSystemFactory@279de282: java.io.IOException: Google Credentials not available, cannot create GCS Under File System.
2019-11-07 21:43:45,136 INFO  ExtensionFactoryRegistry - Loading extension jars from /opt/alluxio/extensions
2019-11-07 21:43:45,138 WARN  UnderFileSystem$Factory - Failed to create UnderFileSystem by factory alluxio.underfs.gcs.GCSUnderFileSystemFactory@53747401: java.io.IOException: Google Credentials not available, cannot create GCS Under File System.
```

**Comment by aniket486** [comment]
Apparently, this is WAI as per:
https://github.com/Alluxio/alluxio/blob/master/underfs/gcs/src/main/java/alluxio/underfs/gcs/GCSUnderFileSystemFactory.java#L71

FYI: @yhqs540

**Comment by medb** [comment]
Related Alluxio issue: https://github.com/Alluxio/alluxio/issues/9917

**Comment by madanadit** [comment]
yep, specifying the credentials explicitly is a known limitation with 2.1. we are working on resolving this in future versions.

**Comment by yhqs540** [comment]
@madanadit Per https://docs.alluxio.io/ee/user/stable/en/ufs/GCS.html, the GCS key is no longer required. But the init action still requires it. The init action also requires `alluxio_download_files_list` property which should not be required for community edition. But the init action fails when the alluxio_download_files_list not specified.

**Comment by Gaurangi94** [comment]
@madanadit Any update on this issue?

**Comment by madanadit** [comment]
@Gaurangi94 it will be fixed in the next release 2.4, as this has been prioritized. It wasn’t fixed in 2.3.

**Comment by Gaurangi94** [comment]
> @Gaurangi94 it will be fixed in the next release, as this has been prioritized. It wasn’t fixed in 2.3.0

Hello, Could you tell me by when should we expecting the next release?

**Comment by medb** [comment]
Alluxio 2.4.0 was just released: https://www.alluxio.io/download/releases/alluxio-2-4-0-release/
